Demo/Application/Base/ContainmentViewController.swift

```diff
@@ -188,7 +188,7 @@ class ContainmentViewController: UIViewController {
 
     private func instantiateViewController(with authorization: String) -> BaseViewController? {
         guard let integrationName = UserDefaults.standard.string(forKey: "BraintreeDemoSettingsIntegration") else {
-            return nil
+            return PayPalWebCheckoutViewController(authorization: authorization)
```

Comment:
PayPal Web made the most sense as the default to me, but if someone feels strongly that we should default to one of the other features, happy to do so!